src/main/java/co/novu/common/base/NovuConfig.java

```diff
@@ -1,16 +1,34 @@
 package co.novu.common.base;
 
+import org.apache.maven.model.Model;
+import org.apache.maven.model.io.xpp3.MavenXpp3Reader;
+
+import java.io.FileReader;
+
 import lombok.Getter;
 import lombok.Setter;
 
 @Getter
 @Setter
 public class NovuConfig {
 
+    private String apiKey;
+    private String baseUrl = "https://api.novu.co/v1/";
+    private String sdkName;
+    private String sdkVersion;
     public NovuConfig(String apiKey) {
         this.apiKey = apiKey;
+        loadContextFromPom();
     }
 
-    private String apiKey;
-    private String baseUrl = "https://api.novu.co/v1/";
+    private void loadContextFromPom(){
+        try {
+            MavenXpp3Reader reader = new MavenXpp3Reader();
+            Model model = reader.read(new FileReader("pom.xml"));
```

Comment:
Reading pom.xml directly will result in reading the pom file of the client app. Please do this instead:
`Model model = reader.read(new InputStreamReader(Objects.requireNonNull(this.getClass().getResourceAsStream("/META-INF/maven/co.novu/novu-java/pom.xml"))));`